src/View/Review/index.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends 'layoutVisitor.html.twig' %}
+
+{% block content %}
+    <div class="container-fluid"><h1>Commentaires clients</h1>
+        {% if server.QUERY_STRING == "success=true&" %}
```

Comment:
ouch c'est moche ça. Je ne connaissais pas comme syntaxe
Non le mieux est d'utiliser soit des session, soit de recup ton GET['success'] dans le controller et de le repasser à ta vue

Reply:
Je ne connais pas cela. Je me renseignerai ce WE. J'ai fait sauter je remodifierai après avoir ajouté la partie ajout de commentaires, puisque c'est plus dans cette US.

Reply:
OK, j'ai préparé le truc comme tu le souhaitais, et l'intègrerai quand la partie form commentaires sera validée.